third-party/gmp/Makefile

```diff
@@ -10,12 +10,15 @@ include $(CHPL_MAKE_HOME)/make/Makefile.base
 HOST_CC=$(shell CHPL_MAKE_HOME=$(CHPL_MAKE_HOME) CHPL_MAKE_HOST_TARGET=--host make -f $(CHPL_MAKE_HOME)/make/Makefile.base printQ-CC)
 
 #
-# Cray X* builds are cross-compilations
+# Cray builds are cross-compilations
 #
 GMP_CROSS_COMPILED=no
 ifneq (, $(filter cray-x% cray-shasta,$(CHPL_MAKE_TARGET_PLATFORM)))
 CHPL_GMP_CFG_OPTIONS += --host=$(CHPL_MAKE_TARGET_ARCH)-cle-linux-gnu
 GMP_CROSS_COMPILED=yes
+else ifneq (, $(filter cray-cs,$(CHPL_MAKE_TARGET_PLATFORM)))
```

Comment:
We probably could use `$(CHPL_MAKE_TARGET_ARCH)-unknown-linux-gnu` for both cray-cs and cray-shasta, and leave cray-xc using 'cle'.  Actually I think we might be able to use the 'unknown' form for all of them, but why mess with success ...

Don't hold the PR for this, just wanted to note it.